src/ServiceControl.Audit.Persistence.RavenDb5/Indexes/AuditCountIndex.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace ServiceControl.Audit.Persistence.RavenDb.Indexes
+{
+    using System.Linq;
+    using Raven.Client.Documents.Indexes;
+    using ServiceControl.Audit.Auditing;
+    using ServiceControl.Audit.Auditing.MessagesView;
+    using ServiceControl.Audit.Monitoring;
+
+    public class AuditCountIndex : AbstractIndexCreationTask<ProcessedMessage, DailyAuditCount>
```

Comment:
LGTM to me from a code perspective. 

My only concern would be making sure that this doesn't impact performance. 

@mauroservienti given the perf test stuff we built during the RavenDB5 TF how much effort would you say it would be to spin up a VM and do a test to verify?

Reply:
Is any of that (how to do those tests) documented?

Reply:
Yes-ish 😬

Reply:
We have https://github.com/Particular/Platform/blob/main/documentation/service-control-load-testing.md and https://github.com/Particular/RavenDB.StatsCollector

Not much but me or @mauroservienti should be able to help out to get you going.

Did we have anything around how to setup the VM mauro?

Reply:
No, but there is nothing special to do. Just install ServiceControl as usual or run it from the command line. Use the load generator with MSMQ to load messages and the stats collector to capture RavenDB stats. Both are command line tools